.github/workflows/.github-actions.yml

```diff
@@ -0,0 +1,25 @@
+name: pyRACF package
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r requirements-development.txt
+    - name: Test with pytest
```

Comment:
We actually don't use the pytest library. We use the built-in unittest library for unit testing. Also, it might make sense to split this test stage into a couple separate stages as follows:
* Lint with flake8 and pylint
* Unit test
* coverage report.

Reply:
Other thing to note here is that `python3 tests/test_runner.py` and `coverage run tests/test_runner.py` are actually doing the same thing. The only difference is that `coverage run tests/test_runner.py` creates a coverage report in binary format. `coverage report -m` just converts the generated coverage report to a readable format. In this case, it prints a test report to the console in text format. So, given that I think you can actually do something like this:

```yaml
   - name: Lint
      run: |
        flake8 .
        pylint --recursive=y .
    - name: Unit Test
        coverage run tests/test_runner.py
    - name: Coverage Report
        coverage report -m
```

Reply:
Agree, this makes sense. Implemented as shown above.